.changeset/small-countries-double.md

```diff
@@ -0,0 +1,25 @@
+---
+'@openfn/language-dhis2': major
+---
+
+- Implement a `util` namespace for non-operation helper functions.
+
+- This means that if you used to do this:
+
+```
+
+findAttributeValueById(state.tei, 'y1w2R6leVmh')
+
+```
+
+do this instead:
+
+```
+
+fn(state =>{
```

Comment:
I can see why you've done this, but in the release notes here, I would exclude the `fn` stuff. Just show a statement-for-statement replacement. Otherwise I think you'll confuse users